scout/constants/ccv.py

```diff
@@ -0,0 +1,244 @@
+# -*- coding: utf-8 -*-
+from collections import OrderedDict
+
+# from worst to most certain benign
+CCV_MAP = OrderedDict(
+    [
+        (4, "oncogenic"),
+        (3, "likely_oncogenic"),
+        (0, "uncertain_significance"),
+        (2, "likely_benign"),
+        (1, "benign"),
+    ]
+)
+# <a href="https://cancerhotspots.org" target="_blank">cancerhotspots.org</a>
+REV_CCV_MAP = OrderedDict([(value, key) for key, value in CCV_MAP.items()])
+
+CCV_OPTIONS = [
+    {"code": "oncogenic", "short": "O", "label": "Oncogenic", "color": "danger"},
+    {
+        "code": "likely_oncogenic",
+        "short": "LO",
+        "label": "Likely Oncogenic",
+        "color": "warning",
+    },
+    {
+        "code": "uncertain_significance",
+        "short": "VUS",
+        "label": "Uncertain Significance",
+        "color": "primary",
+    },
+    {"code": "likely_benign", "short": "LB", "label": "Likely Benign", "color": "info"},
+    {"code": "benign", "short": "B", "label": "Benign", "color": "success"},
+]
+
+CCV_COMPLETE_MAP = OrderedDict([(option["code"], option) for option in CCV_OPTIONS])
+
+CCV_CRITERIA = OrderedDict()
+
+CCV_CRITERIA["oncogenicity"] = OrderedDict(
+    [
+        (
+            "Very Strong",
+            OrderedDict(
+                [
+                    (
+                        "OVS1",
+                        {
+                            "short": "Null variant in tumor supressor",
+                            "description": "Null variant (nonsense, frameshift, canonical ±1 or 2 splice sites, initiation codon, single-exon or multiexon deletion) in a bona fide tumor suppressor gene.",
+                            "documentation": 'Strength can be modified based on <a href="https://pubmed.ncbi.nlm.nih.gov/30192042/" target="blank">ClinGen’s recommendations for PVS1</a>',
```

Comment:
I dont think that was intentionally kept? 
```suggestion
```

Reply:
Actually, it was kept there on the suggestion of a doctor I've been working with. Personally I don't know enough to have an opinion on it